Specs/DateTime/English/DateTimeModelExtendedTypes.json

```diff
@@ -0,0 +1,277 @@
+[{
+  "Input": "schedule me a meeting next week Mon 9 am or 1 pm",
+  "Context": {
+    "ReferenceDateTime": "2017-12-04T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "next week mon 9 am",
+      "TypeName": "datetimeV2.datetimealt",
+      "ParentText": "next week mon 9 am or 1 pm",
+      "Resolution": {
+        "values": [
+          {
+            "timex": "2017-12-11T09",
+            "type": "datetimealt",
+            "value": "2017-12-11 09:00:00"
+          }
+        ]
+      }
+    },
+    {
+      "Text": "1 pm",
+      "TypeName": "datetimeV2.datetimealt",
+      "ParentText": "next week mon 9 am or 1 pm",
+      "Resolution": {
+        "values": [
+          {
+            "timex": "2017-12-11T13",
+            "type": "datetimealt",
+            "value": "2017-12-11 13:00:00"
+          }
+        ]
+      }
+    }
+  ]
+},
+{
+  "Input": "schedule me a meeting next week Mon or Tue",
+  "Context": {
+    "ReferenceDateTime": "2017-12-04T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "next week mon",
+      "TypeName": "datetimeV2.datetimealt",
+      "ParentText": "next week mon or tue",
+      "Resolution": {
+        "values": [
+          {
+            "timex": "2017-12-11",
+            "type": "datetimealt",
+            "value": "2017-12-11"
+          }
+        ]
+      }
+    },
+    {
+      "Text": "tue",
+      "TypeName": "datetimeV2.datetimealt",
+      "ParentText": "next week mon or tue",
+      "Resolution": {
+        "values": [
+          {
+            "timex": "2017-12-12",
+            "type": "datetimealt",
+            "value": "2017-12-12"
+          }
+        ]
+      }
+    }
+  ]
+},
+{
+  "Input": "schedule me a meeting in the morning 9 olock or 10 oclock",
+  "Context": {
+    "ReferenceDateTime": "2017-12-04T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "in the morning 9",
+      "TypeName": "datetimeV2.datetimealt",
+      "ParentText": "in the morning 9 olock or 10 oclock",
+      "Resolution": {
+        "values": [
+          {
+            "timex": "T09",
+            "type": "datetimealt",
+            "value": "09:00:00"
+          }
+        ]
+      }
+    },
+    {
+      "Text": "10 oclock",
+      "TypeName": "datetimeV2.datetimealt",
+      "ParentText": "in the morning 9 olock or 10 oclock",
+      "Resolution": {
+        "values": [
+          {
+            "timex": "T10",
+            "type": "datetimealt",
```

Comment:
The types here are not fully correct. Different tests cases return alternative times, dates, and time ranges. Types should be more specific.